### ISSUE TITLE
Plugin no longer works due to API deprecation

### ISSUE THREAD

**Issue by msbentley** [benji300/joplin-note-tabs#48]
See [this post](https://discourse.joplinapp.org/t/important-plugins-that-use-deprecated-apis-will-no-longer-work/19278). The plugin no longer loads in the latest dev version of Joplin (2.2.4) because of this.

The console shows:
```
/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/Logger.js:188 11:10:29: joplin.plugins: Uncaught exception in plugin "joplin.plugin.note.tabs": Error: "joplin.plugin.note.tabs": No longer supported: joplin.settings.registerSetting() is deprecated in favour of joplin.settings.registerSettings() (deprecated since version 1.8)
    at Plugin.deprecationNotice (/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/Plugin.js:117)
    at JoplinSettings.<anonymous> (/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/api/JoplinSettings.js:85)
    at Generator.next (<anonymous>)
    at /tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/api/JoplinSettings.js:8
    at new Promise (<anonymous>)
    at __awaiter (/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/api/JoplinSettings.js:4)
    at JoplinSettings.registerSetting (/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/api/JoplinSettings.js:84)
    at Object.<anonymous> (/tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/utils/executeSandboxCall.js:50)
    at Generator.next (<anonymous>)
    at /tmp/.mount_Joplin3o96lm/resources/app.asar/node_modules/@joplin/lib/services/plugins/utils/executeSandboxCall.js:8
```

**Comment by benji300** [comment]
Thanks for reporting. I've already changed the implementation on a branch. Hopefully I can create a new version next week.

**Comment by benji300** [comment]
Fixed with latest release v1.4.0 (#50)
